cmd/wallet-js-sdk/test/specs/oidc/presentation/openid4vp.spec.js

```diff
@@ -0,0 +1,185 @@
+} from "@";
+import { IssuerAdapter } from "../../mocks/adapters";
+
+const VC_ISSUER = "vc-issuer-agent";
+const PRC_DESCRIPTOR_ID = "prc_output";
+const VC_FORMAT = "ldp_vc";
+const USER_ID = "agent";
+const MOCK_REQUEST_URL =
+  "openid-vc://?request_uri=https://someverifierdomain.com/v1.0/verifiablecredentials/request/a0eed079-672f-4055-a4f5-e0f5d76ecdea";
+const MOCK_JWT =
+  "eyJhbGciOiJFUzI1NiIsImNydiI6IlAtMjU2Iiwia2lkIjoiZGlkOmtleTp6RG5hZVMzTnA3TUdWSzNodDU5UzZWNUFKU3B1YXI0cWQ5Z1IzVDVUUE5Sb2pBQzR1IiwidHlwIjoiSldUIn0.CgkJewoJCSAgImNsYWltcyI6IHsKCQkJInZwX3Rva2VuIjogewoJCQkgICJpZCI6ICIyMmM3NzE1NS1lZGYyLTRlYzUtOGQ0NC1iMzkzYjRlNGZhMzgiLAoJCQkgICJpbnB1dF9kZXNjcmlwdG9ycyI6IFsKCQkJCXsKCQkJCSAgImlkIjogIjIwYjA3M2JiLWNlZGUtNDkxMi05ZTlkLTMzNGU1NzAyMDc3YiIsCgkJCQkgICJzY2hlbWEiOiBbCgkJCQkJewoJCQkJCSAgInVyaSI6ICJodHRwczovL3d3dy53My5vcmcvMjAxOC9jcmVkZW50aWFscyNWZXJpZmlhYmxlQ3JlZGVudGlhbCIKCQkJCQl9CgkJCQkgIF0sCgkJCQkgICJjb25zdHJhaW50cyI6IHsKCQkJCQkiZmllbGRzIjogWwoJCQkJCSAgewoJCQkJCQkicGF0aCI6IFsKCQkJCQkJICAiJC5jcmVkZW50aWFsU3ViamVjdC5mYW1pbHlOYW1lIgoJCQkJCQldCgkJCQkJICB9CgkJCQkJXQoJCQkJICB9CgkJCQl9CgkJCSAgXQoJCQl9CgkJICB9CgkJfQoJ.o6ldJYTHMbmr7xhqaI-CznY8k9yNjQT7Rhzd4Ns_iBYg6Maj4qhiN7mKEFS9drWbj9v_bN65G44c0qyYcpYJEA";
+const MOCK_DID_DOC = {
+  "@context": ["https://w3id.org/did/v1"],
+  id: "did:key:zDnaeS3Np7MGVK3ht59S6V5AJSpuar4qd9gR3T5TPNRojAC4u",
+  verificationMethod: [
+    {
+      controller: "did:key:zDnaeS3Np7MGVK3ht59S6V5AJSpuar4qd9gR3T5TPNRojAC4u",
+      id: "did:key:zDnaeS3Np7MGVK3ht59S6V5AJSpuar4qd9gR3T5TPNRojAC4u#zDnaeS3Np7MGVK3ht59S6V5AJSpuar4qd9gR3T5TPNRojAC4u",
+      publicKeyJwk: {
+        kty: "EC",
+        crv: "P-256",
+        x: "F-3jdnJCK-McATvRmGEp3trVUmvh4MDGxzmFrvFksLY",
+        y: "gcxVa38yvNnIZKUt12AbLS2IgANk0MvUaGTvnXmHL_Q",
+      },
+      type: "JsonWebKey2020",
+    },
+  ],
+  authentication: [
+    "did:key:zDnaeS3Np7MGVK3ht59S6V5AJSpuar4qd9gR3T5TPNRojAC4u#zDnaeS3Np7MGVK3ht59S6V5AJSpuar4qd9gR3T5TPNRojAC4u",
+  ],
+  assertionMethod: [
+    "did:key:zDnaeS3Np7MGVK3ht59S6V5AJSpuar4qd9gR3T5TPNRojAC4u#zDnaeS3Np7MGVK3ht59S6V5AJSpuar4qd9gR3T5TPNRojAC4u",
+  ],
+  capabilityDelegation: [
+    "did:key:zDnaeS3Np7MGVK3ht59S6V5AJSpuar4qd9gR3T5TPNRojAC4u#zDnaeS3Np7MGVK3ht59S6V5AJSpuar4qd9gR3T5TPNRojAC4u",
+  ],
+  capabilityInvocation: [
+    "did:key:zDnaeS3Np7MGVK3ht59S6V5AJSpuar4qd9gR3T5TPNRojAC4u#zDnaeS3Np7MGVK3ht59S6V5AJSpuar4qd9gR3T5TPNRojAC4u",
+  ],
+  keyAgreement: [
+    {
+      controller: "did:key:zDnaeS3Np7MGVK3ht59S6V5AJSpuar4qd9gR3T5TPNRojAC4u",
+      id: "did:key:zDnaeS3Np7MGVK3ht59S6V5AJSpuar4qd9gR3T5TPNRojAC4u#zDnaeS3Np7MGVK3ht59S6V5AJSpuar4qd9gR3T5TPNRojAC4u",
+      publicKeyJwk: {
+        kty: "EC",
+        crv: "P-256",
+        x: "F-3jdnJCK-McATvRmGEp3trVUmvh4MDGxzmFrvFksLY",
+        y: "gcxVa38yvNnIZKUt12AbLS2IgANk0MvUaGTvnXmHL_Q",
+      },
+      type: "JsonWebKey2020",
+    },
+  ],
+  created: "2022-10-24T18:36:55.219888-04:00",
+  updated: "2022-10-24T18:36:55.219888-04:00",
+};
```

Comment:
nice to put them into `testdata` directory

Reply:
https://github.com/trustbloc/agent-sdk/tree/main/cmd/wallet-js-sdk/test/fixtures/testdata